libs/openFrameworks/graphics/ofBitmapFont.h

```diff
@@ -22,7 +23,7 @@ class ofBitmapFont{
 	~ofBitmapFont();
 	ofMesh getMesh(const string & text, int x, int y, ofDrawBitmapMode mode=OF_BITMAPMODE_MODEL_BILLBOARD, bool vFlipped=true) const;
 	const ofTexture & getTexture() const;
-	ofRectangle getBoundingBox(const string & text, int x, int y) const;
+	ofRectangle getBoundingBox(const string & text, int x, int y, ofDrawBitmapMode mode = ofGetStyle().drawBitmapMode, bool vFlipped = ofIsVFlipped()) const;
```

Comment:
... to clarify "looks funny" .. to call a member variable of a global instance as a default parameter in a function.  Typically I avoid non-constant default parameters because they are ... not constant :)